akka-actor-tests/src/test/scala/akka/util/TypedMultiMapSpec.scala

```diff
@@ -59,12 +59,12 @@ class TypedMultiMapSpec extends WordSpec with Matchers with TypeCheckedTripleEqu
     }
 
     "reject invalid insertions" in {
-      val m1 = TypedMultiMap.empty[AbstractKey, KV]
+      TypedMultiMap.empty[AbstractKey, KV]
```

Comment:
Removing the `m1` here, `"m1.removed(Key(1))(MyValue(42L))" shouldNot compile` below will be trivially true... perhaps we should pull the `val m1 = TypedMultiMap.empty[AbstractKey, KV]` into the string that `shouldNot compile`?